app/javascript/components/base/base-table-component.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <table class="min-w-full divide-y divide-gray-200">
+    <thead class="justify-between">
+      <tr class="bg-yellow-500 border-4 border-yellow-500 py-1">
+        <th v-for="h in table.header">
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Elements in iteration expect to have 'v-bind:key' directives  vue/require-v-for-key